lesson11/main.js

```diff
@@ -0,0 +1,69 @@
+const ul = document.getElementById("js-ul");
+
+const addLoading = () =>{
+  const loading = document.getElementById("js-loading");
+  const loadingImg = document.createElement('img');
+  loadingImg.src = 'loading.gif';
+  loadingImg.id = "loading-img";
+  loading.appendChild(loadingImg);
+};
+
+const removeLoading = () =>{
+  document.getElementById('loading-img').remove();
+};
+
+const renderLists = (features) =>{
+  const fragment = new DocumentFragment();
+
+  for (let feature of features ){
+      const li = document.createElement('li');
+      const a = document.createElement('a');
+      const img = document.createElement('img');
+      a.href = feature.to;
+      a.textContent = feature.text;
+      img.src = feature.img;
+      img.alt = feature.alt;
+
+      fragment.appendChild(li).appendChild(a).insertAdjacentElement('afterbegin', img);
+    };
+
+    ul.appendChild(fragment);
+};
+
+const displayError = (error) =>{
```

Comment:
IMO
This way you should be able to tell the job just by looking at the function name☺️
```suggestion
const InsertErrorStatementInUl = (error) =>{
```

Reply:
Thank you for your suggestion,
I chanaged the name of the variable.

Here is my latest commit 
96057c52090197748e4cabff8d120d3c726ddf63